src/py/reactpy/reactpy/_option.py

```diff
@@ -16,8 +23,9 @@ class Option(Generic[_O]):
     def __init__(
         self,
         name: str,
-        default: _O | Option[_O],
+        default: _O = UNDEFINED,
         mutable: bool = True,
+        parent: Option[_O] | None = None,
         validator: Callable[[Any], _O] = lambda x: cast(_O, x),
```

Comment:
The default value for the validator argument is not appropriate. It should be a callable that returns the validated value.

Reply:
I'm not too concerned about this. If you'd like, we could make this parameter required in a follow-up PR. Would be happy to accept that contribution.